src/System.Management.Automation/engine/Attributes.cs

```diff
@@ -896,6 +896,32 @@ public ValidateLengthAttribute(int minLength, int maxLength) : base()
         }
     }
 
+    /// <Summary>
+    /// predefined range kind to use with 
+    /// ValidateRangeAttribute
+    /// </Summary>
+    public enum ValidateRangeKind{
+        
+        /// <Summary>
+        /// Range is greater than 0
+        /// </Summary>
+        Positive,
+        
+        /// <Summary>
+        /// Range is greater than or equal to 0
+        /// </Summary>
+        NonNegative,
+        
+        /// <Summary>
+        /// Range is less than 0
+        /// </Summary>
+        Negative,
+
+        /// <Summary>
+        /// Range is less than or equal to 0
+        /// </Summary>        
+        NonPositive
```

Comment:
My Vote is for 

1 - Positive, Negative, NonPostive, NonNegative
2 - ValueGTZero, ValueGEZero, ValueGTZero, ValueGEZero